apps/admin-x-settings/src/components/settings/growth/Network.tsx

```diff
@@ -1,22 +1,69 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import TopLevelGroup from '../../TopLevelGroup';
+import validator from 'validator';
 import {Icon, SettingGroupContent, Toggle, withErrorBoundary} from '@tryghost/admin-x-design-system';
+import {Setting, getSettingValues, useEditSettings} from '@tryghost/admin-x-framework/api/settings';
+import {getGhostPaths} from '@tryghost/admin-x-framework/helpers';
+import {useGlobalData} from '../../providers/GlobalDataProvider';
+import {useHandleError} from '@tryghost/admin-x-framework/hooks';
+import {useLimiter} from '../../../hooks/useLimiter';
 
 const Network: React.FC<{ keywords: string[] }> = ({keywords}) => {
-    const enableToggle = (
-        <>
-            <Toggle
-                checked={true}
-                direction='rtl'
-                onChange={() => {}}
-            />
-        </>
-    );
+    const {settings} = useGlobalData();
+    const {mutateAsync: editSettings} = useEditSettings();
+    const handleError = useHandleError();
+    const limiter = useLimiter();
+
+    // The Network toggle is disabled in Admin settings if:
+    // 1. (Ghost (Pro) only) the feature is disabled by config
+    // 2. The site is hosted on a subdirectory, localhost or an IP address in production
+    const [isDisabledByConfig, setIsDisabledByConfig] = useState(false);
+    useEffect(() => {
+        async function checkLimiter() {
+            if (limiter?.isLimited('limitSocialWeb') && await limiter.checkWouldGoOverLimit('limitSocialWeb')) {
```

Comment:
Curious why we chose to use `checkWouldGoOverLimit()` here instead of `isDisabled()`?